config.js

```diff
@@ -87,6 +87,30 @@ export default {
           "ios/fastlane/metadata/review_information/review_demo_user.txt"
         ],
         upgradeManifestImport: "./manifest/210-to-211.json"
+      },
+      {
+        text: "Upgrade my scaffold (2.1.1 -> 2.2.0)",
+        previousVersion: "2.1.1",
+        previousVersionSHA: "67204d92e9b460edc36099ebdd6c2cd691c48a57",
+        nextVersion: "2.2.0",
+        nextVersionSHA: "99ae211777fe0ae4ae0bd3a601297ad1beb590cb",
+        ignoreTemplatize: [
+          "ios/fastlane/metadata/review_information/review_demo_password.txt",
+          "ios/fastlane/metadata/review_information/review_demo_user.txt"
+        ],
+        upgradeManifestImport: "./manifest/211-to-220.json"
+      },
+      {
+        text: "Upgrade my scaffold (2.2.0 -> 2.3.0)",
+        previousVersion: "2.2.0",
+        previousVersionSHA: "99ae211777fe0ae4ae0bd3a601297ad1beb590cb",
+        nextVersion: "2.3.0",
+        nextVersionSHA: "PLAT-11232-create-validation",
+        ignoreTemplatize: [
+          "ios/fastlane/metadata/review_information/review_demo_password.txt",
+          "ios/fastlane/metadata/review_information/review_demo_user.txt"
+        ],
+        upgradeManifestImport: "./manifest/220-to-230.json"
```

Comment:
This doesn't look right, you need to rebase your PR. Only one new version should be added.